src/generateRenderer/breaks.js

```diff
@@ -0,0 +1,71 @@
+const Classifier = require('classybrew')
+
+module.exports = { createBreaks }
+
+function createBreaks (features, classification) {
+  let values
+  try {
+    values = getValues(features, classification.classificationField)
+  } catch (e) {
+    console.log(e)
+    return
+  }
+  if (classification.breakCount > values.length) classification.breakCount = values.length // make sure there aren't more breaks than values
+
+  return calculateBreaks(values, features, classification)
+    .map((value, index, array) => { return [array[index - 1] || array[0], value] }).slice(1)
+}
+
+function getValues (features, field) {
+  // TODO: should featureCollection metadata fields be checked too?
+  return features.map((feature, index) => {
+    const properties = feature.properties || feature.attributes // TODO: should this conditional be an option?
+    const key = Object.keys(properties).filter(property => { return property === field })
+    const value = Number(properties[key])
+    if (isNaN(value)) throw new TypeError('Cannot use values from non-numeric field')
+    return value
+  })
+}
+
+function calculateBreaks (values, features, classification) {
+  const normValues = normalizeValues(values, features, classification)
```

Comment:
Probably. It is running every time with a check at the beginning with a quick return if there is no normalization option. I will move this outside the function